plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -213,19 +212,6 @@ def fn_partial():
         else:
             raise IgnoreOutputs()
 
-    def execute(self, **kwargs) -> Any:
-        """
-        This method will be invoked to execute the task.
-
-        Handles the exception scope for the `_execute` method.
-        """
-        from flytekit.exceptions import scopes as exception_scopes
-
-        if self.execution_mode == self.ExecutionBehavior.DEFAULT:
-            return exception_scopes.user_entry_point(self._execute)(**kwargs)
```

Comment:
@kumare3 I think we do need to wrap the execution with `exception_scopes.user_entry_point()`. I first didn't have this and every exception in user code appeared to lead to a retry even if `retries=0`. If I understand correctly, because without this any user-code exception is counted against the system failures retry budget?

Reply:
I agree, I was going to refactor the base class to make execute the leaf method so that we won't have to do this again. 
But we'll let it be, put the if block and do it again.
Sorry my bad

Reply:
https://github.com/flyteorg/flytekit/pull/1603/commits/4b2ce0c1d95db731b3e691f203dc47c788ee2d62

This should work as well right?
I tested by running an elastic task from within a dynamic subworkflow.

The if/else is only because both `@task` and `@dynamic` inherit from this class?